### PR TITLE
[twine-sugarcube] update to 2.33 and fixes

### PR DESCRIPTION
**Pull request by ezsh**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/tmedwards/sugarcube-2/issues/45#issuecomment-663843123
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)

### CODE REVIEW

File: types/twine-sugarcube/wiki.d.ts [hunk 1]
```diff
@@ -1,22 +1,15 @@
...
-     */
-    isExternalLink(link: string): boolean;
-    parse(text: string): any;
-    /**
-     * @see State.setVar
-     */
-    setValue(varName: string, value: any): boolean;
+    new(destination: OutputDestination | null, source: string, options?: WikifierOptions): unknown;
+
+	createExternalLink(destination: OutputDestination, url: string, text: string): HTMLElement;
```
Comment (danger-public) [inline types/twine-sugarcube/wiki.d.ts:10]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

File: types/twine-sugarcube/jquery.d.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Passage } from "./passage";
+
+declare global {
+	namespace JQuery {
```
Comment (danger-public) [inline types/twine-sugarcube/jquery.d.ts:4]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Comment by danger-public** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **types/twine-sugarcube/test/events.ts#L2** - The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d30c2fe6800819c0f3bc7a33ea278278feeb7c7b
</p>

File: types/twine-sugarcube/test/events.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+$(document).on(":passageinit", event => {
+	event.passage; // $ExpectType Passage
```
Comment (danger-public) [inline types/twine-sugarcube/test/events.ts:2]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

File: types/twine-sugarcube/jquery.d.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            extends PassageNavigationEvent<TDelegateTarget, TData, TCurrentTarget, TTarget> {
+            /**
+             * The element which holds (or will be holding in case of the ':passageinit' event) the redered passage.
+             */
+            content: HTMLElement;
+        }
+
+        interface TypeToTriggeredEventMap<TDelegateTarget, TData, TCurrentTarget, TTarget> {
+            ':passageinit': PassageNavigationEvent<TDelegateTarget, TData, TCurrentTarget, TTarget>;
+			':passagestart': PassageRenderingEvent<TDelegateTarget, TData, TCurrentTarget, TTarget>;
```
Comment (danger-public) [inline types/twine-sugarcube/jquery.d.ts:23]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Comment by kant2002** [comment]
@tmedwards can you take a look?

**Comment by tmedwards** [comment]
Looks good to me. 👍

**Review by tmedwards — APPROVED** [review]
👍

**Comment by tmedwards** [comment]
Ready to merge

**Pull request merged**